### DISCUSSION TITLE
wslview on Oracle Linux 9

### DISCUSSION THREAD

**Discussion by hsabillon7** [wslutilities/wslu#306] [Q&A]
I have installed wslu version 3.2.4-26.1 in Oracle Linux 9 and I'm trying to get a web browser open for authentication but instead, opens a file explorer window. Tried directly with wslview https://google.com with the same result. Maybe I'm missing some config?

**Comment by car-meows** [comment]
The latest wslu is 4.x. How do you install the packages?
